### DISCUSSION TITLE
Can we set "remaining work" on save estimate?

### DISCUSSION THREAD

**Discussion by thiagofonsecaNS** [microsoft/azure-boards-estimate#50] [Ideas]
Can we set "remaining work" on save estimate?

**Comment by byamagata** [comment]
What would be the use case for this?
Are you adding remaining work at the Bug/User Story/PBI level?  Or are you talking about on tasks?

**Reply by thiagofonsecaNS** [reply]
About tasks @byamagata . When estimate about task finish we need to enter in the task card and fill this field.

**Comment by byamagata** [comment]
Ahhh ok, Ive never used this tool with tasks so I couldnt understand.
Is this tool meant to be used with tasks?  Generally poker is used for
refinement and planning meetings when talking about Bugs and User Stories,
no?

On Mon, Nov 29, 2021 at 11:56 AM Thiago Araujo da Fonseca <
***@***.***> wrote:

> About tasks @byamagata <https://github.com/byamagata> . When estimate
> about task finish we need to enter in the task card and fill this field.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/azure-boards-estimate/discussions/50#discussioncomment-1717824>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGHEZT7NEIQC4RW2AFFN5JLUOPEFPANCNFSM5I7FR4AQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Benny Yamagata Jr.
Software Engineer
NetDocuments

**Reply by thiagofonsecaNS** [reply]
Makes sense. I may be doing this wrong! Thank you @byamagata
